packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.tsx

```diff
@@ -0,0 +1,211 @@
+import type { CSSObject } from 'styled-components';
+import type {
+  BaseBoxProps,
+  MakeValueResponsive,
+  SpacingValueType,
+  ArrayOfMaxLength4,
+} from './types';
+import type { Breakpoints } from '~tokens/global/breakpoints';
+import breakpoints from '~tokens/global/breakpoints';
+import { getMediaQuery } from '~src/utils/getMediaQuery';
+import { isReactNative, isEmpty, getIn, makeSpace, makeSize } from '~utils';
+import type { Theme } from '~components/BladeProvider';
+
+const getResponsiveValue = <T extends string | number | string[]>(
```

Comment:
Could you also add few test cases for these utils? Would be good for maintainability.

Reply:
done 🎉 found some nice enhancements and some bugs while writing test 👯